builder/osc/common/artifact.go

```diff
@@ -127,3 +130,49 @@ func (a *Artifact) stateAtlasMetadata() interface{} {
 
 	return metadata
 }
+
+// stateHCPPackerRegistryMetadata will write the metadata as an hcpRegistryImage for each of the AMIs
+// present in this artifact.
+func (a *Artifact) stateHCPPackerRegistryMetadata() interface{} {
+
+	f := func(k, v interface{}) (*registryimage.Image, error) {
+
+		region, ok := k.(string)
+		if !ok {
+			return nil, fmt.Errorf("unexpected type of key in Omis map")
+		}
+		imageId, ok := v.(string)
+		if !ok {
+			return nil, fmt.Errorf("unexpected type for value in Omis map")
+		}
+		image := registryimage.Image{
+			ImageID:        imageId,
+			ProviderRegion: region,
+			ProviderName:   "osc",
+		}
+
+		return &image, nil
+
+	}
+
+	images, err := registryimage.FromMappedData(a.Omis, f)
```

Comment:
should we use Images instead of Omis ?
also I would use OMIs, as OMI mean Outscale Machine Image IIRC, so it's an acronym.